### PR TITLE
feat: randomize hiring cost of brothers between +-20%

### PR DESCRIPTION
**Pull request by Darxo**
I briefly tested this ingame and on the first town it worked fine.

### CODE REVIEW

**Comment by LordMidas** [comment]
I feel this is rather unnecessary as hiring cost is already varied by equipment. But I don't see any reason to **not add** this. So, I don't have any strong opinions about it. Please get a review from @Enduriel about the code and then feel free to merge.

**Review by LordMidas — APPROVED** [review]

File: mod_reforged/config/math/math.nut [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        return _value;
+    }
+
+    // Private helper functions
+    function getCloser( _oldValue, _newValue, _target)
+    {
+        local oldDistance = ::Math.abs(_oldValue - _target);
+        local newDistance = ::Math.abs(_newValue - _target);
+        return (newDistance < oldDistance) ? _newValue : _oldValue;
+    }
```
Comment (Enduriel) [inline mod_reforged/config/math/math.nut:32]: Should be double underscored to identify as a private function

File: mod_reforged/config/math/math.nut [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+			}
+		}
+		return _randomNumber;
+	}
+
+	// A negative _decimalPlaceOffset will result in an equal amounts of 0's left of the decimal place.
+	// ceil(1234, -2) = 1300; ceil(1234, -1) = 1240; ceil(1234.5678, 2) = 1234.57;
+	function ceil( _value, _decimalPlaceOffset = 0 )
+	{
+		_value *= ::Math.pow(10, _decimalPlaceOffset);
```
Comment (LordMidas) [inline mod_reforged/config/math/math.nut:21]: We should benchmark the built-in squirrel `pow` function against this `::Math.pow`

Comment (LordMidas) [reply mod_reforged/config/math/math.nut:21]: I did some benchmark and `::Math.pow` is slightly slower. So I'd say just use `pow`.

**Pull request merged**